src/codecs/elife/elife.test.ts

```diff
@@ -0,0 +1,48 @@
+import { ElifeCodec } from '.'
+import * as vfile from '../../util/vfile'
+import unlink from '../../util/unlink'
+import { nockRecord, snapshot } from '../../__tests__/helpers'
+import { YamlCodec } from '../yaml'
+
+const { decode, sniff, encode } = new ElifeCodec()
+const yaml = new YamlCodec()
+
+const elife2yaml = async (article: string) => {
+  // Fetch, with recording the complete article
+  const done = await nockRecord(`nock-record-${article}.json`)
+  const node = await decode(await vfile.load(`elife: ${article}`))
+  done()
+  // Unlink to remove references to local files (which are non-deterministric)
+  const unlinked = await unlink(node)
+  // Convert to YAML
+  return await vfile.dump(await yaml.encode(unlinked))
+}
+
+
+jest.setTimeout(30 * 1000)
+
+test('sniff', async () => {
```

Comment:
A suggestion for future PR, but I'd recommend not grouping a lot of `expect` calls like this, especially when they're testing different bits of logic (like whitespace in some of the cases below).
Reason being that when a test fails, it's much harder to know why, and which expect call was the culprit.
Instead using a `describe` block, with multiple `test` cases named after the scenarios they're testing can be clearer and easier to debug.